website/src/content/deployment/production-build.mdx

```diff
@@ -2,7 +2,12 @@
 title: Production Build
 ---
 
-After you've begun developing your application, it's time to think about deployment for testing and production purposes. This section introduces what you need to do to prepare your application for production deployment, and provides deployment examples.
+After authoring your Custom Application, you eventually want to deploy your application to a staging environment for testing purposes or to a production environment.
+One of the key steps to prepare your Custom Application for a deployment is creating a production build of your Custom Application.
+
+This section helps prepare your Custom Application for deployment by creating a production build.
+
+# Build
```

Comment:
Maybe we can mention the prerequisite section here as well, like in the compile app page.
The only prerequisite here would be `[Configure your Custom Application](/development/application-config).`

Reply:
good call